packages/services/api/src/modules/oidc-integrations/providers/oidc-integrations.provider.ts

```diff
@@ -237,6 +253,20 @@ export class OIDCIntegrationsProvider {
       } as const;
     }
 
+    const currentUser = await this.session.getViewer();
+    await this.auditLog.record({
+      eventType: 'ORGANIZATION_SETTINGS_UPDATED',
+      updatedFields: JSON.stringify({
+        updateOIDCIntegration: true,
+      }),
```

Comment:
TODO @TuvalSimha, here we can put in updated fields.

Reply:
Do you want to store all args?

something like that:
```

      eventType: 'OIDC_INTEGRATION_UPDATED',
      updatedFields: JSON.stringify({
        clientId: clientIdResult.success ? args.clientId : null,
        clientSecret: clientSecretResult.success ? args.clientSecret : null,
        tokenEndpoint: tokenEndpointResult.success ? args.tokenEndpoint : null,
        userinfoEndpoint: userinfoEndpointResult.success ? args.userinfoEndpoint : null,
        authorizationEndpoint: authorizationEndpointResult.success ? args.authorizationEndpoint : null,
        
      }),
```

@n1ru4l